### PR TITLE
Update README.en.md

### PR DESCRIPTION
**Pull request by VulcanSphere**
Wording, guidelines -> methods per another pending pull request

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -67,7 +68,7 @@ You can use [bebasid batch script](https://github.com/bebasid/bebasid/releases/t
 
 ### <ins>Other</ins>
 
-Please head to [USAGE](https://github.com/bebasid/bebasid/blob/master/dev/readme/USAGE.md) for more usage guidelines.
+Please head to [USAGE](https://github.com/bebasid/bebasid/blob/master/dev/readme/USAGE.md) for more usage methods.
```
Comment (nxzlem) [inline README.en.md:71]: I think for `more usage methods` needs a verb fix, like `for other methods`.

**Review by nxzlem — APPROVED** [review]
Overall, it's fine; you have to fix what I commented on the different lines. If you want to merge, go ahead as long as it looks fine to you.

File: README.en.md [hunk 2]
```diff
@@ -35,6 +35,7 @@
   - [Graphical User Interface (GUI)](#graphical-user-interface-gui)
   - [Terminal User Interface (TUI)](#terminal-user-interface-tui)
   - [Command Line Interface (CLI)](#command-line-interface-cli)
+  - [Other](#other)
```
[the remark this answers is not in the document]
Comment (VulcanSphere) [reply README.en.md:38]: Good suggestion, thanks

File: README.en.md (hunk 1 above)
Comment (VulcanSphere) [reply README.en.md:71]: Good suggestion

**Pull request merged**